lab-adan/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const greet = require('./lib/greet.js');
+const arithmetic = require('./lib/arithmetic.js');
+
+const output = module.exports = () => {
```

Comment:
You don't actually need to export anything from this file because this is the entry file which other files/modules get exported to. From this file you can simply call the methods you required in.